app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        return Distance(
+            self.km + self.number_return(other)
+        )
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        self.km += self.number_return(other)
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        if isinstance(other, (int, float)):
+            return Distance(
+                self.km * self.number_return(other)
+            )
+
+    def __truediv__(self, other: float | int) -> Distance:
+        if isinstance(other, (int, float)):
```

Comment:
This check is not needed
The same in mul
Because we already say that we expect only float or int